server/src/pages/shotindex/view.js

```diff
@@ -167,6 +190,12 @@ class Body extends React.Component {
   }
 
   onOpen(url, event) {
+    if (event.target.id === "toggle-share" ||
+        event.target.classNames === "alt-actions-contaner") {
```

Comment:
@johngruen, 
There is a small typo on this line.
I believe "alt-actions-contaner" should be "alt-actions-container".